internal/network/utils.go

```diff
@@ -0,0 +1,38 @@
+package network
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"encoding/json"
+
+	"github.com/openshift/assisted-service/models"
+)
+
+func IsIpv6OnlyHost(host *models.Host, log logrus.FieldLogger) bool {
+	if host.Inventory == "" {
```

Comment:
Shouldn't this be an error? If there's not enough info, it doesn't mean that the host isn't IPv6-only. It looks like inventory info if expected here, so maybe we need to prevent calling the function without it (i.e. return an error).

Reply:
Missing inventory is not an error

Reply:
My point is that it's not an indication of non-IPv6-only either. What does missing inventory info mean?

Reply:
That the node is in discovering state